zebra-state/src/error.rs

```diff
@@ -34,16 +38,40 @@ pub struct CommitBlockError(#[from] ValidateContextError);
 /// An error describing why a block failed contextual validation.
 #[derive(displaydoc::Display, Debug, Error)]
 #[non_exhaustive]
+#[allow(missing_docs)]
 pub enum ValidateContextError {
-    /// block.height is lower than the current finalized height
+    /// block height {candidate_height:?} is lower than the current finalized height {finalized_tip_height:?}
```

Comment:
if we're going to include a lot of interpolated values we should probably not use `Displaydoc`
```suggestion
    #[error("block height {candidate_height:?} is lower than the current finalized height {finalized_tip_height:?}")]
```

Reply:
Ok, I'll do this fix quickly now

Reply:
Fixed in 50b66e01b0a175bfaebde46a6f94d396d309f8d0